index.html

```diff
@@ -11,74 +11,132 @@
-    <div class="banner-one">
-      <h2>0zł</h2>
-      <p><b>za prowadzenie konta, kartę i wypłaty ze wszystkich bankomatów,</b></br> jeśli aktywnie korzystasz z konta osobistego, czyli przelewasz na nie swój dochód (min. 1000zł wpływów zewnętrznych miesięcznie) i </br> płacisz min. raz w miesiącu kartą do konta lub BLIKIEM</p>
+  </nav>
+
+  <header class="hero__container">
+    <div class="wrapper hero__content">
+      <h1 class="hero__title">Konto 360° pełne korzyści</h1>
+      <span class="hero__hint">z aplikacją mobilną na każdą pogodę</span>
+      <button class="btn btn--hero hero__btn">Załóż konto 360° &gt;</button>
     </div>
-  </article>
-  <article id="banner bottom">
-    <div class="banner-two">
-      <div class="first-col">
-        <h1>0zł</h1>
-        <p>za przelewy krajowe w złotych w Millenecie i aplikacji mobilnej</p>
-      </div>
-      <div class="second-col">
-        <h1>0zł</h1>
-        <p>za przelewy na e-mail lub telefon</p>
-      </div>
-      <div class="third-col">
-        <h1>0zł</h1>
-        <p>za realizację zleceń stałych i poleceń zapłaty</p>
+  </header>
+
+  <section id="about" class="about__container section section--about">
+    <div class="section__header section__header--about"><!-- !!! byla spacja w ID !!! -->
+      <div class="wrapper">
+        <h2 class="about__title section__title">O koncie</h2>
       </div>
     </div>
-  </article>
-  <p class="app-p1">Aplikacja mobilna na każdą pogodę</p>
-  <p class="app-p2">Pobierz i aktywuj bezpłatną aplikację mobilną Banku Millennium, by mieć jeszcze więcej możliwości:</p>
-  <article id="banner app">
-    <div class="app">
-      <div class="circle ticket">
-        <img src="img/parking.png">
-        <p><b>Kupisz bilet autobusowy i opłacisz parking</b></br> w każdej chwili, bez rejestracji i dodatkowych opłat</p>
-      </div>
-      <div class="circle money">
-        <img src="img/money.png">
-        <p><b>Weźmiesz pożyczkę</b></br> w kilka kliknięć, bez zbędnych formalnośc</p>
+
+    <div class="section__content section__content--about">
+      <div class="wrapper">
+        <h3 class="about__subtitle section__subtitle">
+          <strong>Korzystaj z konta za 0 zł</strong> w dowolnym miejscu i czasie
+        </h3>
+      
+        <div class="price__container price__container--main">
+          <p class="price__content">
+            <span class="price__value price__value--main">0zł</span>
+            <span class="price__description">
+              <strong>za prowadzenie konta, kartę i wypłaty ze wszystkich bankomatów,</strong></br> 
+              jeśli aktywnie korzystasz z konta osobistego, czyli przelewasz na nie swój dochód (min. 1000zł wpływów zewnętrznych miesięcznie) i płacisz min. raz w miesiącu kartą do konta lub BLIKIEM
+            </span>
+          </p>
+        </div>
+
+        <div class="about__minor-prices">
+
+            <div class="price__container price__container--transfers">

```

Comment:
price__container - szerokosc kolumny - klasa ogolna dla wszytkich kolumn, zeby nie pisac tak jak bylo col-1 col-2 i col-3 - po co skoro sa rowne. A jesli nie sa - to roznice opisujemy w modyfikatorach - np `--transfers`. 

Col-1 col-2 i col-3 nie mowia nic o tresci - kolejnosc moze sie zmienic i co wtedy? :P